src/renderer/components/inputs/SliderInput.tsx

```diff
@@ -122,6 +130,17 @@ const SliderInput = memo(
                             <NumberDecrementStepper />
                         </NumberInputStepper>
                     </NumberInput>
+                    <Text
+                        fontSize="xs"
+                        m={0}
+                        mb={-1}
+                        mt={-1}
+                        p={0}
+                        pb={-1}
+                        pt={-1}
```

Comment:
Negative padding values are invalid and therefore ignored.

Reply:
I actually just copied this code from NodeBody, so I'll fix it there too.